src/assert.js

```diff
@@ -1,9 +1,14 @@
+import { pauseProcessing, resumeProcessing, objectValues } from "./core";
```

Comment:
it will convert es6 to CommonJS modules, we would need to run Browserify after that in order to bundle them.